libr/util/str.c

```diff
@@ -3972,3 +3972,19 @@ R_API bool r_str_startswith(const char *str, const char *needle) {
 	return !strncmp (str, needle, strlen (needle));
 }
 
+
+R_API const char *r_str_seek(const char *s, int n) {
```

Comment:
```suggestion
R_API const char *r_str_skip(const char *s, int n) {
```

probably a more appropriate name since seek could imply support for negative arg